Automattic/wp-calypso
# SPT: Adds WP eslint rules

Adds @wordpress eslint rules to the Plugin.

Currently there is a problem in that eslint is running from the root calypso dir and so cannot find the plugin using the syntax

`extends: [ 'plugin:@wordpress/eslint-plugin/recommended' ],`

in `apps/full-site-editing/.eslintrc.js`

Perhaps some kind of absolute path but I'm not sure how to do this.

#### Testing instructions

* Make change. Try to commit. See linter run.
* Run `npm run lint:js:spt` from `apps/full-site-editing` dir.

*

Fixes #
